src/guides/v2.2/javascript-dev-guide/widgets/widget_modal.md

```diff
@@ -152,6 +176,41 @@ Sets a margin between slide modal windows.
 
 **Default value**: 45
 
+### `modalSubTitle` {#modal_modalSubTitle}
+The modal subtitle element selector.
```

Comment:
Could I kindly ask you to provide more information about the subtitle? Particularly, what is the difference between title and subtitle?

 Thank you!

Reply:
Actually the purpose of  `subTitle` was described for another option were we set the text for subTitle:  `subTitle`.
https://github.com/magento/devdocs/pull/6118/files#diff-ee454d3c5298af295b7f3eef90f0da47R229
Please let me know if we need to adjust it somehow.

Thank you.

Reply:
Got it, missed that point. Thank you